starter-code/app.js

```diff
@@ -2,4 +2,74 @@ const express = require('express');
 const app = express();
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
+const bodyParser = require('body-parser');
 
+
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+app.get('/', (req, res, next) => {
+
+  res.render("index");
+
+});
+
+app.get("/random", (req, res, next)=> {
+  client.getRandomJoke()
+  .then((response) => {
+    // use the response here
+    console.log(response);
+    res.send(`<p> ${response.value}</p>`);
+  }).catch((err) => {
+    // handle error
+  });
+
+
+});
+
+
+app.get("/categories", (req, res, next)=> {
+
+  client.getJokeCategories()
+  .then((response)=>  {
+    // use the response here
+    res.render('categories', {res: response});
```

Comment:
here you have a chance to write the es6 syntax -> {response} instead of {res: response}